src/components/FeedbackComponent.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const FeedbackComponent = () => {
+    return (
+        <div>
+            Not finding the answer you're looking for? Share your feedback on these docs by creating an issue in our <a href="https://github.com/LIT-Protocol/Issues-and-Reports/issues">GitHub Issues and Reports</a> repository or get support by visiting our <a href="/v3/support/intro">Support</a> page. We are here to help you!
```

Comment:
can we make this a callout box (like the one at the top of this page: https://developer.litprotocol.com/v3/what-is-lit) just so it stands out a bit more?

i think we can also remove the end 'we are here to help you' text

Reply:
Nice suggestion @a1ttech!
Fixed :)